Fixes some wordings around system index permission access and usage (#4948)
opensearch-project/documentation-website: _security/access-control/permissions.md

{: .tip }

> ## System Index permissions

Reviewer: ```suggestion
## System index permissions
```

System index permissions are unique among other permissions in that they extend some traditional admin-only accessibility to non-admin users. These permissions give normal users the ability to modify any system index specified in the role or roles to which they are mapped. The exception to this is the security system index, `.opendistro_security`, which is used to store Security's configuration YAML files and remains accessible only to admins with an admin certificate.

> System index permissions are specified under `index_permissions` in the `roles.yml` configuration file (See [roles.yml]({{site.url}}{{site.baseurl}}/security/configuration/yaml/#rolesyml).) and can be granted permission to as any regular index by specifying the action `system:admin/system_index` under `allowed_actions`. Here is an example, the `.opendistro-alerting-config` system index stores configurations for the Alerting plugin. 

Author: I don't think the system index permissions always begin with `.opendistro`, i know they begin with `.` but that is more of a convention than a constraint.

> Along with standard index permissions, system index permissions are specified in the 'roles.yml' configuration file under 'index_permissions' (See [roles.yml](site.urlsite.baseurl/security/configuration/yaml/#rolesyml). The user gains access to the system index by specifying `system:admin/system_index` in the role's `allowed_actions` section. With the addition of this action, all system indices covered by patterns defined in `index_patterns` are now accessible to this role. The `*` pattern, on the other hand by itself, is an exception that cannot be used to grant access to all system indices at once.

Reviewer: ```suggestion
Along with standard index permissions, you specify system index permissions in the 'roles.yml' configuration file under 'index_permissions' (See [roles.yml](site.urlsite.baseurl/security/configuration/yaml/#rolesyml). You do this by specifying `system:admin/system_index` in the role's `allowed_actions` section and adding the system index in the `index_patterns` section. 
```

"The `*` pattern, on the other hand by itself, is an exception that cannot be used to grant access to all system indices at once." is already stated below this in the "following considerations" bullet points. Let's say this here or there, but not both places.

Author: aah, good catch, i will use your suggestion here.

The following example shows a way to grant permission system index permission specified in a role called `alerting-main-role`:

```yml

> alerting-main-role:

Author: even better. my understanding was it might have been a typo.
